packages/app-defaults/src/defaults/apis.ts

```diff
@@ -259,6 +261,20 @@ export const apis = [
       });
     },
   }),
+  createApiFactory({
```

Comment:
Tbh I think it'd be best to implement this as something that's completely built-in in the sign-in page instead, rather than having a Utility API for it. There's no point calling this API outside the sign-in page right?

Reply:
Yup, what would be a good way to ensure people have the module installed? Currently it's all frontend and doesn't require a backend installation.